<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgEditTopologia</class>
 <widget class="QDialog" name="dlgEditTopologia">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tipología</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Esquema Estrcuctural</string>
       </property>
       <property name="buddy">
        <cstring>cboEsquemasEstructurales</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>2</number>
       </property>
       <item>
        <widget class="QComboBox" name="cboEsquemasEstructurales"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnDetailsEsquemas">
         <property name="text">
          <string>Detalles...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>2</number>
       </property>
       <item>
        <widget class="QComboBox" name="cboSecciones"/>
       </item>
       <item>
        <widget class="QPushButton" name="btnDetailsSection">
         <property name="text">
          <string>Detalles...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Seccion</string>
       </property>
       <property name="buddy">
        <cstring>cboSecciones</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Solicitaciones</string>
       </property>
       <property name="buddy">
        <cstring>btnAddSolicitacion</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="btnAddSolicitacion">
       <property name="text">
        <string>Nueva...</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Material</string>
       </property>
       <property name="buddy">
        <cstring>cboMaterial</cstring>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QListWidget" name="listSolicitaciones"/>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="cboMaterial"/>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Método de Cálculo</string>
       </property>
       <property name="buddy">
        <cstring>cboMetodoCalculo</cstring>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="cboMetodoCalculo"/>
     </item>
     <item row="7" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="btnDiagramas">
         <property name="text">
          <string>Diagramas</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnArmadura">
         <property name="text">
          <string>Armaduras...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cboEsquemasEstructurales</tabstop>
  <tabstop>btnDetailsEsquemas</tabstop>
  <tabstop>cboSecciones</tabstop>
  <tabstop>btnDetailsSection</tabstop>
  <tabstop>btnAddSolicitacion</tabstop>
  <tabstop>listSolicitaciones</tabstop>
  <tabstop>cboMaterial</tabstop>
  <tabstop>cboMetodoCalculo</tabstop>
  <tabstop>btnDiagramas</tabstop>
  <tabstop>btnArmadura</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>dlgEditTopologia</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>443</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>dlgEditTopologia</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>311</x>
     <y>443</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
